File: help/eng_gb/exercicios_da_ficha_7/Ficha07_002/Ficha07_002.cpp
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <iostream>
#include <cstring>
#include <cmath>
using namespace std;

// Class definition
class Ficha07_002 {

    public:
        int main() {
            cout << "Summation step 2\n";
            int sum  = 0 ;
            int i  = 0 ;
            for( int i = 1; i < 10; i = i + 2 ) {
                cout << sum  << " + " << i ;
                sum +=i;
                cout << " \t= " << sum  << "\n";
            }
            cout << "\nSum = " << sum ;
            return 0;
        }
};// end of class Ficha07_002
// Execute main function
int main(){
    Ficha07_002 application; // create object
    application.main(); // execute object
    return 0;
}

//C++  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Fri, 2022 Sep 30 22:36

File: help/eng_gb/exercicios_da_ficha_7/Ficha07_002/Ficha07_002.cs
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

using System.IO;
using System;
public class Ficha07_002 {

    //Main Function
    static void Main() {
        Console.Write("Summation step 2\n" );
        int sum = 0 ;
        for( int i = 1; i < 10; i = i + 2 ) {
            Console.Write(sum + " + " + i );
            sum +=i;
            Console.Write(" \t= " + sum + "\n" );
        }
        Console.Write("\nSum = " + sum );
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Fri, 2022 Sep 30 22:36
